src/clients/authorization/ImportDialog.tsx

```diff
@@ -0,0 +1,156 @@
+          data-testid="cancel"
+          id="modal-cancel"
+          key="cancel"
+          variant={ButtonVariant.link}
+          onClick={() => {
+            closeDialog();
+          }}
+        >
+          {t("common:cancel")}
+        </Button>,
+      ]}
+    >
+      <Form>
+        <JsonFileUpload id="import-resource" onChange={setImported} />
+      </Form>
+      {Object.keys(imported).length !== 0 && (
+        <>
+          <Divider />
+          <p className="pf-u-my-lg">{t("importResources")}</p>
+          <Form isHorizontal>
+            <FormGroup
+              label={t("policyEnforcementMode")}
+              labelIcon={
+                <HelpItem
+                  helpText="clients-help:policyEnforcementMode"
+                  forLabel={t("policyEnforcementMode")}
+                  forID="policyEnforcementMode"
+                />
+              }
+              fieldId="policyEnforcementMode"
+              hasNoPaddingTop
+            >
+              <Radio
+                id="policyEnforcementMode"
+                name="policyEnforcementMode"
+                label={t(
+                  `policyEnforcementModes.${imported.policyEnforcementMode}`
+                )}
+                isChecked
+                isDisabled
+                className="pf-u-mb-md"
+              />
+            </FormGroup>
+            <FormGroup
+              label={t("decisionStrategy")}
+              labelIcon={
+                <HelpItem
+                  helpText="clients-help:decisionStrategy"
+                  forLabel={t("decisionStrategy")}
+                  forID="decisionStrategy"
+                />
+              }
+              fieldId="decisionStrategy"
+              hasNoPaddingTop
+            >
+              <Radio
+                id="decisionStrategy"
+                name="decisionStrategy"
+                isChecked
+                isDisabled
+                label={t(`decisionStrategies.${imported.decisionStrategy}`)}
+                className="pf-u-mb-md"
+              />
+            </FormGroup>
+            <FormGroup
+              hasNoPaddingTop
+              label={t("allowRemoteResourceManagement")}
+              fieldId="allowRemoteResourceManagement"
+              labelIcon={
+                <HelpItem
+                  helpText={t("allowRemoteResourceManagement")}
+                  forLabel={t("allowRemoteResourceManagement")}
+                  forID={"allowRemoteResourceManagement"}
+                />
+              }
+            >
+              <Switch
+                id="allowRemoteResourceManagement"
+                label={t("common:on")}
+                labelOff={t("common:off")}
+                isChecked={imported.allowRemoteResourceManagement}
+                isDisabled
+              />
+            </FormGroup>
+          </Form>
+          <div className="pf-u-mt-md">
+            {Object.entries(imported)
+              .filter((entry) => Array.isArray(entry[1]))
```

Comment:
You could also destructure this.
```suggestion
              .filter(([, value]) => Array.isArray(value))
```